.github/workflows/ci-repo.yaml

```diff
@@ -0,0 +1,55 @@
+name: Konveyor CI repo testing for PRs and pushes to main
+
+on:
+  push:
+    branches:
+      - "main"
+
+  pull_request:
+    branches:
+      - "main"
+
+concurrency:
+  group: ci-repo-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-images-to-test-loading:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pull some images from docker or quay.io and save them to a tar file
+        run: |
+          docker pull docker.io/library/busybox:latest
+          docker pull quay.io/fedora/fedora:latest
+          docker pull registry.access.redhat.com/ubi9/nodejs-20-minimal:latest
+
+          mkdir -p /tmp/images/a /tmp/images/b
+
+          docker save -o /tmp/images/busybox.tar docker.io/library/busybox:latest
+          docker save -o /tmp/images/a/fedora.tar quay.io/fedora/fedora:latest
+          docker save -o /tmp/images/b/nodejs.tar registry.access.redhat.com/ubi9/nodejs-20-minimal:latest
```

Comment:
Could you provide more background why we want use those non-konveyor images? Thanks.

Reply:
This is just to test that one or more container images can be put in an artifact and successfully loaded into the minikube docker registry.  The contents of the container don't really matter, I just chose these 3 since they're all coming from different public registries.

Plus, non-konveyor images won't effect the actual running of the tests.

A "real" test would also then be able to verify the containers loaded here actually load in tests -- just no real good way to do that without a bunch more work.

Reply:
Plus I wanted to have something in place to test out #86 (well VISACK it at least)